### ISSUE TITLE
possible bug in date calculations?

### ISSUE THREAD

**Issue by dave256** [lemonmade/templates#23]
I have a $startDate variable which I enter in a dialog as Jan 11 2016

and then I have due dates for items set as:
due: $dateOfStart+1w+3d 5pm

and it makes them due on Friday, not Thursday. Since Jan 11 2016 is a Monday I would expect that +1w+3d to be Thursday. So it's off by one day or am I doing something wrong and I should be using +1w+2d 5PM to get Thursday?

**Comment by lemonmade** [comment]
Yeah, the script doesn't support adding dates and setting a time in the same date. The reason it goes to Friday is that the addition above resolves to +1w 2d 17h (5PM = 17 hours from midnight). This obviously isn't a great way to handle it, but unfortunately, I don't have much time these days to work on the date math.

Sorry for the inconvenience!

**Comment by dave256** [comment]
No problem. I can easily work around it by doing one less day. If you're not going to fix it, I'll just do that but I didn't want to make all those changes if you were going to fix it soon. My AppleScript knowledge is minimal so I won't attempt to fix it either. I'll let you choose whether to leave the issue open or closed.
